tests/algorithms/test_mce_irl.py

```diff
@@ -314,6 +314,31 @@ def test_tabular_policy(rng):
     np.testing.assert_equal(timesteps[0], 2 - mask.astype(int))
 
 
+def test_tabular_policy_rollouts(rng):
+    """Tests tabular policy prediction, especially timestep calculation and masking."""
```

Comment:
What the test is doing is a bit hard to follow, consider expanding docstring, adding comments or otherwise clarifying?

My understanding (but it took a few minutes for me to work out) is:
  - `s_a` always takes the same action (deterministically) regardless of state, and alternates round-robin between the three actions.
  - `pi` is `s_a` tiled 7 times to get up to 21 timesteps, greater than the horizon of `20` of `ReasonablePOMDP`.
  - We then check that the actions taken in fact do alternate round robin as we'd expect from the construction.

Reply:
Updated - hopefully it's clearer now